spec/tictac_spec.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/logic'
+
+describe Game do
+  let(:new_game) { Game.new }
+  let(:player) { Player.new('Deyan', 'X') }
+  describe '#make_move' do
+    it 'check if it is correct' do
+      expect(new_game.make_move(player.team, 7)). to eql false
+    end
+    it 'Check if it is a string' do
+      expect(new_game.make_move(player.team, 'string')).to eql false
+    end
+    it 'check if the number is from 1 to 9' do
+      expect(new_game.make_move(player.team, 12)).to eql false
+    end
+  end
+
+  describe '#check_if_win?' do
+    it 'Check if there is no winner' do
+      expect(new_game.check_if_win?).to eql false
+    end
```

Comment:
Be sure to check if there is a winner. For each of your methods, you have to write tests both positive and negative scenarios.

Reply:
I have problem-solving this issue.